### PR TITLE
Py module

### PR DESCRIPTION
**Pull request by amdrozdov**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package:
```
Comment (dkorolev) [inline Makefile:7]: Need `.PHONY:` for these (I can add).

File: MANIFEST.in [hunk 2]
```diff
@@ -0,0 +1,2 @@
+recursive-include pls/static/.pls *
```
Comment (dkorolev) [inline MANIFEST.in]: Why not just `pls/static/`?

File: pls/functions.py [hunk 3]
```diff
@@ -0,0 +1,515 @@
...
+
+    def add_to_queue(src_dir):
+        abs_src_dir = os.path.abspath(src_dir)
+        queue_list.append(abs_src_dir)
+        queue_set.add(abs_src_dir)
+
+    add_to_queue(src_dir)
+    while queue_list:
+        src_dir = queue_list.popleft()
+        if src_dir not in already_traversed_src_dirs and (
```
Comment (dkorolev) [inline pls/functions.py]: Why break lines? Isn't there some `cols: 120` option in the tool you use for `fmt`?

File: pylama.ini [hunk 4]
```diff
@@ -0,0 +1,8 @@
+[pylama]
+format = pylint
+skip = */.tox/*,*/.env/*
+linters = pylint,mccabe
+
+ignore = C901
+[flake8]
+max-line-length = 120
```
Comment (dkorolev) [inline pylama.ini:8]: I see there is -- so looks wrong; maybe you added it later?

File: setup.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+def get_reqs(file="requirements.txt"):
+    with open(file) as f:
+        return [s.rstrip("\n") for s in f.readlines()]
+
+setup(
+    name="pls",
+    version="0.1",
+    packages=find_packages(),
+    install_requires=get_reqs(),
+    author="Dmitry Korolev",
```
Comment (dkorolev) [inline setup.py]: Dima pls =)

File: setup.py [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    author_email="dmitry.korolev@gmail.com",
```
Comment (dkorolev) [inline setup.py]: dima@current.ai pls =)

File: tests/test_smoke.py [hunk 7]
```diff
@@ -0,0 +1,5 @@
+from pls.functions import cmd_version
+
+def test_smoke():
+    cmd_version(None)
+
```
Comment (dkorolev) [inline tests/test_smoke.py]: Blank line

**Review by dkorolev — APPROVED** [review]
Minor comments. Ty && LGTM!

File: pls/functions.py (hunk 3 above)
Comment (dkorolev) [reply pls/functions.py]: https://chatgpt.com/share/0c6db2a5-9a0f-4cff-a412-34d0708bc24c =)

File: MANIFEST.in (hunk 2 above)
Comment (dkorolev) [reply MANIFEST.in]: Decided to rename it into some `dotpls`.

Comment (amdrozdov) [reply MANIFEST.in]: rename to `dot_pls`

File: pylama.ini (hunk 4 above)
Comment (dkorolev) [reply pylama.ini:8]: https://chatgpt.com/share/0c6db2a5-9a0f-4cff-a412-34d0708bc24c =)

File: pls/functions.py (hunk 3 above)
Comment (amdrozdov) [reply pls/functions.py]: change black line-length=120

**Comment by amdrozdov** [comment]
Done. Updated all + tested locally

**Review by dkorolev — APPROVED** [review]

**Pull request merged**